### PR TITLE
IGVF-721-create-Auxiliary-FileSet

### PR DESCRIPTION
**Pull request by iyoungworth**
mappings

pluralizing

insert-fix

### CODE REVIEW

File: src/igvfd/schemas/auxiliary_set.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            "type": "string",
+            "enum": [
+                "gRNA sequencing",
+                "oligo-conjugated lipids",
+                "oligo-conjugated antibodies"
+            ]
+        },
+        "construct_libraries": {
+            "title": "Construct Libraries",
+            "description": "The construct libraries of vectors introduced to the biosample used to produce data in this measurement set.",
```
Comment (gabdank) [inline src/igvfd/schemas/auxiliary_set.json:66]: this measurment_set or auxiliary_set?

File: src/igvfd/schemas/auxiliary_set.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            "uniqueItems": true,
+            "items": {
+                "title": "Construct Library",
+                "type": "string",
+                "linkTo": "ConstructLibrary"
+            }
+        },
+        "moi": {
+            "title": "Multiplicity Of Infection",
+            "description": "The actual multiplicity of infection (MOI) for vectors introduced to the biosample used to produce data in this measurement set. At least one construct library must be specified in order to specify MOI. This property should capture the actual MOI, and not the targeted MOI.",
```
Comment (gabdank) [inline src/igvfd/schemas/auxiliary_set.json:79]: measurment_set or auxiliary_set?

File: src/igvfd/searches/configs/auxiliary_set.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+from snovault.elasticsearch.searches.configs import search_config
```
Comment (gabdank) [inline src/igvfd/searches/configs/auxiliary_set.py:1]: need to add auxiliary_set to the defaultsearch list

**Review by gabdank — CHANGES_REQUESTED** [review]
see comments in PR

File: src/igvfd/searches/configs/auxiliary_set.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+@search_config(
+    name='AuxiliarySet'
+)
+def cell_line():
```
Comment (gabdank) [inline src/igvfd/searches/configs/auxiliary_set.py:7]: I don't think it is cell_line

**Review by gabdank — CHANGES_REQUESTED** [review]
minor typo in the search/config

File: src/igvfd/searches/configs/auxiliary_set.py (hunk 4 above)
Comment (iyoungworth) [reply src/igvfd/searches/configs/auxiliary_set.py:7]: Sorry about this obvious copy-paste error, fixed

**Review by gabdank — APPROVED** [review]

**Pull request merged**